spec/features/profile_spec.rb

```diff
@@ -1,23 +1,32 @@
 require 'rails_helper'
 
-feature 'Edit profile' do
-  let(:user) { create(:user) }
+RSpec.feature 'Edit profile' do
   let(:features) { FeaturesSpecHelper.new }
```

Comment:
This object is a bit confusing, because it's not part of the test, but is only here for a setup purpose.
It's not a good idea to have such background object, put on the foreground. This disturb the reader, and force him to focus on your setup, instead of on the very behaviour of the test.

My advice: refactor your FeaturesSpecHelper, in order to make it as transparent as possible, like this:

```
module Features::ConnectionHelpers
  def sign_in(user)
    [...]
  end
end
```

With this one, you would be able to sign in from your test in one line, like this:

`background { sign_in(user) }`